frontend/src/components/WhyUs/style.module.css

```diff
@@ -0,0 +1,77 @@
+.container {
+  height: 91vh;
+  background-color: var(--background-gradient);
+  display: grid;
+  grid-template-columns: 50% 50%;
+  grid-template-areas: "content vectors";
+}
+.content {
+  grid-area: content;
```

Comment:
We are using two layout systems: `flex` and `grid`. We should be consistent and use just one. Create an issue to refactor code to use just one.

Reply:
I don't think using flexbox and grid in the same app is an issue. They both have some similarities but both are good for different use cases.

I agree that this component can be built using flexbox but I don't think it's worth changing it.